CHANGELOG.md

```diff
@@ -0,0 +1,112 @@
+## 2018-04-18, Version 0.3.0
+### Commits
+- [[`650df0bf4d`](https://github.com/yoshuawuyts/human-panic/commits/650df0bf4de62239e9592b2185ebbd4875640864)] (cargo-release) version 0.3.0 (Yoshua Wuyts)
+- [[`a3ec0ddb97`](https://github.com/yoshuawuyts/human-panic/commits/a3ec0ddb97eb5940ab56785eb54eed52ead3a82b)] Fixing wrong data in certain crash dump fields (#15)
+
+Fixing a bug in the log dumps
+
+* Carrying over metadata from first macro call to properly
+include metadata in crash dumps.
+* Making Method derive Copy
 (Katharina)
```

Comment:
Oops oops, it looks like this file was written using an old version of `changelog`. There was a bug that created these `^M` instances which has since been fixed. Could you perhaps try updating and running this again?

Reply:
I don't have a `changelog` installed? How is this tool invoked?